packages/ui-components/src/__tests__/DeploymentSteps.test.ts

```
@@ -1,30 +1,43 @@
-	getAccount: vi.fn()
-}));
 
 vi.mock('$lib/hooks/useGui', () => ({
 	useGui: vi.fn()
 }));
 
+vi.mock('$lib/providers/wallet/useAccount', () => ({
+	useAccount: vi.fn()
+}));
+
 export type DeploymentStepsProps = ComponentProps<DeploymentSteps>;
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)